pallets/asset-manager/src/weights.rs

```diff
@@ -52,17 +52,19 @@ use sp_std::marker::PhantomData;
 
 /// Weight functions needed for pallet_asset_manager.
 pub trait WeightInfo {
-	fn register_asset() -> Weight;
+	fn register_foreign_asset() -> Weight;
 	fn set_asset_units_per_second(x: u32) -> Weight;
 	fn change_existing_asset_type(x: u32) -> Weight;
 	fn remove_supported_asset(x: u32) -> Weight;
+	fn authorize_local_assset() -> Weight;
+	fn register_local_asset() -> Weight;
 }
 
 /// Weights for pallet_asset_manager using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	fn register_asset() -> Weight {
-		(52_375_000 as Weight)
+	fn register_foreign_asset() -> Weight {
+		(43_113_000 as Weight)
```

Comment:
nothing changed in this one besides the name right?

Reply:
that's right, I will leave it as it was